lib/plugins/aws/provider/awsProvider.js

```diff
@@ -145,14 +145,73 @@ class AwsProvider {
             type: 'string',
             pattern: '^arn:',
           },
+          awsCfFunction: {
+            oneOf: [
+              { $ref: '#/definitions/awsCfImport' },
+              { $ref: '#/definitions/awsCfJoin' },
+              { $ref: '#/definitions/awsCfGetAtt' },
+              { $ref: '#/definitions/awsCfRef' },
+              { $ref: '#/definitions/awsCfSub' },
+            ],
+          },
           awsCfImport: {
             type: 'object',
             properties: {
-              'Fn::ImportValue': { type: 'string' },
+              'Fn::ImportValue': {},
```

Comment:
This schema needs to stay as is, as in our internal logic we rely on fact that it's a _string_ (unconditionally), e.g. it's the case here:

https://github.com/serverless/serverless/blob/b0938c7d9bd946c5c9af6bae99ae6f7931242ba6/lib/plugins/aws/info/getStackInfo.js#L37

If AWS allows non string values here, we probably should have both `awsCfImport`, and `awsCfImportLocallyResolvable`, as suggested in this comment: https://github.com/serverless/serverless/pull/8112#discussion_r474594677